bionic/decorators.py

```diff
@@ -440,6 +441,31 @@ def returns(out_descriptor):
     )
 
 
+def aip_resource(resource):
+    """
+    Indicates that the decorated function should be computed in AIP.
+    This decorator requires AIP based distributed execution to be enabled, which
+    can be done by setting ``core__aip_execution__enabled`` core entity.
+
+    Parameters
+    ----------
+    resource: bionic.aip.task.Resource
```

Comment:
Assuming users aren't going to use the `Resource` class for anything else, would it make more sense to just accept  arguments like `machine_type` and so on directly instead of making them instantiate a `Resource`?

Reply:
I think I would need to change this anyway. @baxen this seems related to [your comment](https://github.com/square/bionic/pull/249#discussion_r488070772) on your recent PR. What fields would you want configurable? I initially thought we would only want to configure the resource, but it seems like I got it wrong.

Reply:
What seems most natural to me is for `@aip_resource` to take a `machine_type` directly, and have the job config be set either via global config entities or (maybe, optionally) via a separate `@aip_job` decorator. This seems equally composable (you can always instantiate a decorator and reuse it in your code) but requires less boilerplate for the common case. @namanjain @baxen WDYT?

Reply:
We talked offline about this. We'll continue keeping the global config the way it is, and explode the class as separate fields on this decorator. This is experimental anyway, so it won't hurt to try one way and change it before we GA.